### DISCUSSION TITLE
libhelix - allocateation failed when AudioPlayer.end()

### DISCUSSION THREAD

**Discussion by AlphaGaming7780** [pschatzmann/arduino-libhelix#23] [Q&A] [answered]
I want to end the audio player, but I get this error `libhelix - allocateation failed for 8708 bytes`.

Somebody know how to fix that ?

Here are the logs:
```
[I] AudioEncoded.h : 169 - virtual void audio_tools::EncodedAudioOutput::end()
[I] MetaDataID3.h : 575 - virtual void audio_tools::MetaDataID3::end()
[I] AudioPlayer.h : 203 - reset codec
libhelix - allocateation failed for 8708 bytes
```

**Comment by pschatzmann** [comment, marked as answer]
I just committed a correction that should address this issue!

**Reply by AlphaGaming7780** [reply]
Thank you, how can I update/test this fix ?

**Reply by pschatzmann** [reply]
It depends on how you installed the library. If it was via git just execute a git pull.
Otherwise delete it and reinstall it...

**Reply by AlphaGaming7780** [reply]
Yep, that works, thank you for that quick fix.

**Reply by nath654nath** [reply]
hey ! i have the same issue... can you help me ?
